Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -640,6 +640,22 @@ class B3IRGenerator {
     }
 
 private:
+    void emitPrepareWasmOperation(BasicBlock* block)
+    {
+#if !USE(BUILTIN_FRAME_ADDRESS) || ASSERT_ENABLED
+        // Prepare wasm operation calls.
+        PatchpointValue* patchpoint = block->appendNew<PatchpointValue>(m_proc, B3::Void, origin());
+        patchpoint->effects = B3::Effects::forCall();
+        patchpoint->append(instanceValue(), ValueRep::SomeRegister);
+        patchpoint->setGenerator([=] (CCallHelpers& jit, const StackmapGenerationParams& params) {
+            GPRReg instanceGPR = params[0].gpr();
+            jit.storePtr(GPRInfo::callFrameRegister, CCallHelpers::Address(instanceGPR, Instance::offsetOfTemporaryCallFrame()));
```

Comment:
ditto

Reply:
actually, I think this entire patchpoint can just be normal B3 IR. Just a Store(FramePointer)

Reply:
Done.